### PR TITLE
chore: Fix mod game version being wrongly published on Modrinth/Curseforge

### PR DESCRIPTION
**Pull request by kristofbolyai**

### CODE REVIEW

**Review by DevScyu — APPROVED** [review]

File: .github/workflows/beta-release.yml [hunk 1]
```diff
@@ -135,6 +135,7 @@ jobs:
           name: Wynntils Artemis (${{ matrix.modloader }}) ${{ needs.changelog.outputs.tag }}
           version: ${{ needs.changelog.outputs.tag }}
           version-type: beta
+          game-versions: 1.20.2
```
Comment (magicus) [inline .github/workflows/beta-release.yml:138]: It would be nice to read this off some gradle config file. It is easy to forget to change this value...

Comment (kristofbolyai) [reply .github/workflows/beta-release.yml:138]: This is a temporary fix most likely, as we didn't need this before.

Comment (magicus) [reply .github/workflows/beta-release.yml:138]: I'm guessing it doesn't hurt to have that information here, if Modrinth starts acting up again... But what will hurt is if we forget to update this number.

Comment (kristofbolyai) [reply .github/workflows/beta-release.yml:138]: I usually do CTRL-Find our old version in the project, when I update to a new version. Hopefully we won't forget, and if we do, we'll fix this :)

**Pull request merged**